E2E-ORS-Sample-For-Java/src/main/java/app.config

```diff
@@ -0,0 +1,14 @@
+sourceContainerName = sourcecontainer
+sourceStorageAccountName = testsourcestorageaccount
+sourceStorageAccountConnectionString = XXX
+
+destinationContainerName = destcontainer
+destinationStorageAccountName = testdeststorageaccount
+destinationStorageAccountConnectionString = XXX
+
+resourceGroup = carmen_romero
```

Comment:
remove your resource group before you merge

Reply:
good to know! thank you